config/install/05-calico-jenkins.yaml

```diff
@@ -93,7 +93,7 @@ spec:
       terminationGracePeriodSeconds: 0
       initContainers:
       - name: flexvol-driver
-        image: quay.io/saurabh/flexvol:latest
+        image: quay.io/saurabh/flexvol:04242018
```

Comment:
don't we want to test against latest in our STs?

Reply:
sure. i did not apply that tag but can do that.